src/main/java/menion/android/whereyougo/gui/activity/CartridgeDetailsActivity.java

```diff
@@ -74,9 +74,7 @@ public void onCreate(Bundle savedInstanceState) {
 
         TextView tvDistance = (TextView) findViewById(R.id.layoutDetailsTextViewDistance);
 
-        Location loc = new Location(TAG);
-        loc.setLatitude(MainActivity.cartridgeFile.latitude);
-        loc.setLongitude(MainActivity.cartridgeFile.longitude);
+        Location loc = new Location(MainActivity.cartridgeFile.latitude, MainActivity.cartridgeFile.longitude);
```

Comment:
The 'TAG' string is now unused. Please remove.

Reply:
Done in 2daea5a655714047859cba8bb48e8b816def4518